README.md

```diff
@@ -126,65 +110,67 @@ Install the required python packages
 python -m pip install -r requirements.txt
 ```
 
-### Telemetry link
 
-**Create a fake Telemetry Gateway**
+## Development
+
 
-Upload `dummy_gateway.ino` to any Arduino board with a USB port. Make sure to uncomment one of these two lines before :
+### Dashboard
 
-```c
-// Uncomment one of these to select the target gateway
-// #define lps
-#define telemetry
+**Run the GUI**
+
+Run `dashboard.py` with a dummy telemetry link
+
+```
+python ./dashboard.py dummy
 ```
 
+Enjoy
 
-**Run the GUI**
 
-Make sure the board is connected to your computer
+### Launch Pad Station control
 
-Make sure `dashboard.py` is set to use the fake gateway
+Get the *Launch Pad Station Board* up and running. See [aesirkth/LaunchPadStation](https://github.com/aesirkth/LaunchPadStation)
 
-```py
-if __name__ == "__main__":
-    # Use this with a RFD900 modem
-    # serial = SerialWrapper(baudrate=115200, name="Telemetry", rfd900=True)
-    # Use this for testing with an Arduino board and `dummy_telemetry.ino`
-    serial = SerialWrapper(baudrate=115200, name="Telemetry", bonjour="TELEMETRY")
-```
 
-Run `dashboard.py`
+**Run the GUI**
+
+Make sure the *Launch Pad Station Board* is connected to your computer
+
+Run `lps_control.py`
 
 ```
-python ./dashboard.py
+python ./lps_control.py

```

Comment:
Take this one away because of repetition?

Reply:
It's not really a repetition the context is different